microsoft/testsuites/kselftest/kselftest-suite.py

```diff
@@ -49,13 +49,14 @@ def verify_kselftest(
     ) -> None:
         file_path = variables.get("kselftest_file_path", "")
         working_path = variables.get("kselftest_working_path", "")
-        run_as_root = variables.get("kselftest_run_as_root", False)
+        install_as_sudo = variables.get("install_as_sudo", True)
```

Comment:
This change should be reverted, the installation doesn't need the configurable sudo.

Reply:
Please revert the change, so the behavior is not changed when running kselftests.